roles/network_plugin/canal/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+# The interface used by canal for host <-> host communication.
+# If left blank, then the interface is chosing using the node's
+# default route.
+canal_iface: ""
+
+# Whether or not to masquerade traffic to destinations not within
+# the pod network.
+canal_masquerade: "true"
```

Comment:
Guys, it's supposed to be a string since we pass it in exactly this way into PoD as ENV variable.